requirements/requirements-dev.txt

```diff
@@ -1,24 +1,22 @@
 #
-# This file is autogenerated by pip-compile with Python 3.9
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
And here I prefer generating it with Python 3.9 as that's the minimum supported version of `safejax`